.config/README.md

```diff
@@ -0,0 +1,70 @@
+# Some Config Files Are Here
+
+In here, config files some.
+
+## TypeScript Configuration
+
+We have three (3) TypeScript configuration files here:
+
+- `tsconfig.workspace.json` - "Base" config for workspaces in this repo.
+- `tsconfig.src.json` - Config for files within a workspace's `src/` directory (sources).
+- `tsconfig.test.json` Config for files within a workspace's `test/` directory (tests).
+
+In addition, we have a `tsconfig.json` in the workspace (project) root, configured for incremental builds. This file maintains references to `tsconfig.json` files in each workspace which needs types to be checked and/or built.
+
+In addition the root `tsconfig.json` serves as the "default" for files in the repo which do not otherwise have an ancestor `tsconfig.json` file.
+
+### How to Configure a Workspace for TypeScript
+
+> For the talented & gifted, you can skip this and copy the setup in e.g., `packages/aa`.
+
+1. Create a `tsconfig.json` file in the workspace and set its contents to:
+
+   ```json
+   {
+     "extends": "../../.config/tsconfig.workspace.json",
+     "references": [
+       { "path": "./src/tsconfig.json" },
+       { "path": "./test/tsconfig.json" }
+     ]
+   }
+   ```
+
+   If you don't want to typecheck your tests, omit the second object in the `references` array.
```

Comment:
I do have mixed feelings about typechecking tests. Mostly because I worry it might get in the way. Happy to give it a spin and push back if it does.

Reply:
It can get in the way, especially when straying off of the happy path.  It may or may not be worth it.  Generally, I think it is.